pxtextmining/factories/factory_pipeline.py

```diff
@@ -293,7 +281,7 @@ def create_sklearn_pipeline(model_type, tokenizer=None, additional_features=True
                     probability=True,
                     class_weight="balanced",
                     max_iter=1000,
-                    cache_size=500,
+                    cache_size=1000,
                 ),
                 n_jobs=-1,
```

Comment:
Does this use all cores? May be a problem on the server (or it may not, I'm really not sure)